src/actions/party.ts

```diff
@@ -1,73 +1,116 @@
-import * as types from './types';
+import { ActionTypes } from './types';
 import Party from '../services/party';
 import { NavigationActions } from 'react-navigation';
 
-export function hostParty() {
-    return (dispatch, getState) => {
-        dispatch(NavigationActions.navigate({ routeName: 'Party' }))
-        const user = getState().user;
-        Party.host(user)
-            .then(party => {
-                dispatch(subscribeToParty(party))
-                return party;
-            })
-            .then(party => dispatch(setPartyRecieved(party)))
-            .catch(e => console.log(e));
-    };
-}
+export class PartyManager {
```

Comment:
Why does it have to be a class at all? These are all the action creators related to the party. They are exported individually so that they can either be connected individually to a component, or to be imported as a hash only to be combined with every other action creator [see here](https://github.com/lycan-city/werewolf-virtual-cards/blob/0d4ebcb9156c8405a3af2f0ebf1c4ed2d2e55853/src/actions/index.ts#L5).

As a matter of fact I think you broke `actions/index` by encapsulating all these actions in a class.